hm_1/tasks/task_02_01.py

```diff
@@ -0,0 +1,72 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+import string
+from collections import Counter
+from typing import List
+
+
+def get_data(file_path: str) -> str:
+    data = ""
+    with open(file_path, encoding="unicode_escape") as f:
+        for line in f:
+            data += line
+    return data
+
+
+def get_longest_diverse_words(file_path: str) -> List[str]:
+    data = {}
+    result = []
+    text_without_punctuation = get_data(file_path).translate(
+        str.maketrans("", "", string.punctuation)
+    )
+    words = text_without_punctuation.split()
+    for i in words:
+        data[i] = len(set(i.lower()))
+    k = len(data) if len(data) <= 10 else 10
```

Comment:
What will happen if you pass k that bigger than the list length to slicing?

Reply:
It will be an error(Index out of Range), but I make condition to avoid this situations, or I don’t see anything?

Reply:
Please provide a code example.